charts/kamaji/README.md

```diff
@@ -68,6 +68,7 @@ Here the values you can override:
 | affinity | object | `{}` | Kubernetes affinity rules to apply to Kamaji controller pods |
 | cfssl.image.repository | string | `"cfssl/cfssl"` |  |
 | cfssl.image.tag | string | `"latest"` |  |
+| clusterDomain | string | `"cluster.local"` | Domain of the Kubernetes cluster. |
```

Comment:
Since it's related to Datastore, may I ask you to rename this to `datastore.clusterDomain`, please?

Reply:
Of course, but the dns domain pertains to all services and pods within a cluster, which is why it is commonly defined as a global variable.

I think one could argue that in this case the cluster domain primarily affects the `etcd` deployment, so that might be a better fit, if it should not be global?

Reply:
I was just focusing on the scope of that variable, since without the Datastore, that would be useless, also considering we're planning to remove the Datastore templating to a dedicated chart (#238).

Furthermore, I think we have to bump the minor version of the Chart in order to release publicly to the community the new feature you implemented.

Reply:
I think the `clusterDomain` primarily affects deployment of the built in etcd cluster, since they are the services which require the fqdn to establish a quorum.

I was not aware of the kamaji-etcd chart, but it looks like I should have been using that all along :)

I think the best course at this point is to close both the issue and this pull request outright, and then I can create a patch for https://github.com/clastix/kamaji-etcd instead.